mne/channels/channels.py

```diff
@@ -1007,11 +1011,52 @@ def _ch_neighbor_connectivity(ch_names, neighbors):
     ch_connectivity = np.eye(len(ch_names), dtype=bool)
     for ii, neigbs in enumerate(neighbors):
         ch_connectivity[ii, [ch_names.index(i) for i in neigbs]] = True
-
     ch_connectivity = sparse.csr_matrix(ch_connectivity)
     return ch_connectivity
 
 
+def compute_connectivity(info, ch_type):
```

Comment:
I agree. I actually had something like ``find_neighbors`` at first, but there is ``read_ch_connectivity`` that basically does the same thing, except it reads this stuff from a file.